Ports/doxygen/package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=doxygen
+version='1.9.7'
+files=(
+    "https://github.com/doxygen/doxygen/archive/refs/tags/Release_${version//./_}.tar.gz 691777992a7240ed1f822a5c2ff2c4273b57c1cf9fc143553d87f91a0c5970ee"
+)
+workdir="${port}-Release_${version//./_}"
+useconfigure='true'
+configopts=(
+    -Bbuild
+    -DICONV_INCLUDE_DIR:PATH="${SERENITY_INSTALL_ROOT}/usr/local/include/iconv.h"
+    -DICONV_LIBRARY:FILEPATH="${SERENITY_INSTALL_ROOT}/usr/local/lib/libiconv.so"
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_POLICY_DEFAULT_CMP0148=OLD
+)
+makeopts+=(
+    -Cbuild
+)
+installopts+=(
+    -Cbuild
+)
+depends=(
+    libiconv
+)
```

Comment:
Please adhere to the quoting as shown in the Ports README.

Reply:
Why do we need to have quotes around depends, makeopts, port name there is no space between them?

Reply:
We just like to make it obvious to the reader that those are strings, and nothing else.

Reply:
Makes sense. I am working on shellcheck tested refactor of ports to include single or double quotes based on variable interpolation.